index.html

```diff
@@ -270,6 +270,43 @@ <h4 class="more_det1 more-de2">LET’S CONNECT</h4>
         </li>
     </form>
   </section>
-  <script src="mobile-menu.js"></script>
+  <div class="popup-window" id="popup">
+    <section class="popup">
+      <div class="popup-header">
+        <h2 class="popup-project-title" id="popup-project-title"></h2>
+        <span class="close">&times;</span>
+      </div>
+      <div class="popup-project-info" id="popup-project-info"></div>
+      <div id="popup-project-image-holder">
+        <img
+          src="#"
+          alt="Tonic project screenshot"
+          class="popup-project-image"
+          id="popup-project-image"
+        />
+      </div>
+      <div class="popup-project-details">
+        <div class="column-left">
+          <p
+            class="popup-project-description"
+            id="popup-project-description"
+          ></p>
+        </div>
+        <div class="column-right">
+          <div class="popup-project-stack" id="popup-project-stack"></div>
+          <div class="popup-project-buttons">
+            <button type="button" id="see-live-btn" onclick="">
+              See Live<img src="./media-icons/Icon.png" alt="media icon"class="media-pop">
+            </button>
+            <button type="button" id="see-source-btn" onclick="">
+              See Source<img src="./media-icons/Vector-pop.png" alt="media icon"class="media-pop">
+            </button>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Same on your pop up window, your contents are too thin to read. Can you please increase the font-size.
- [ ] In addition, your buttons should link to your project live version and your project's GitHub source code

screenshot |
:--------------:|
![c2](https://user-images.githubusercontent.com/39523312/151284661-10c0fa6c-5a5e-4080-9ef6-a7dce6c86f27.png)